version/version.go

```diff
@@ -0,0 +1,14 @@
+package version
+
+import "runtime/debug"
+
+var Commit = func() string {
```

Comment:
Can you document this exported variable? Some interesting things to say about it:
- Why this is needed / when it should be used
- An example of how it can be used (e.g. a log method call)